draft-irtf-panrg-path-properties.md

```diff
@@ -124,6 +124,13 @@ Assessed property:
 : An approximate calculation or assessment of the value of a property. An assessed property includes the reliability of the calculation or assessment. The notion of reliability depends on the property.
 For example, a path property based on an approximate calculation may describe the expected median one-way latency of packets sent on a path within the next second, including the confidence level and interval. A non-numerical assessment may instead include the likelihood that the property holds.
 
+## Terminology usage for specific technologies
+
+The terminology described in this document is intended to be general and applicable to existing and future path-aware technologies.
+This document attempts to unify terminology which is used inconsistently between different path-aware technologies.
```

Comment:
I'm not sure about this sentence. Maybe we can delete it?

The introduction already gives the motivation for the common terminology:
"As terms related to paths have different meanings in different areas of networking, first, this document provides a common terminology to define paths, path elements, and path properties."

And I'm not sure if we really unify existing terminology - we did survey existing terminology, but "unify" sounds like a stronger implication that what the doc actually does, and I'm not sure if we even "attempt to unify".

If we want to add motivation to this section, how about adding the following to the first sentence above?
"[…] to provide a common language for the Path-Aware Networking Research Group."
Because I think that's actually the main motivation for having a generic common terminology that applies to different areas of networking.

Reply:
I like your suggestion better as well, let's change it.

Reply:
```suggestion
```